profiles/common/modules/custom/tmgmt_og/tmgmt_og.module

```diff
@@ -115,6 +115,24 @@ function tmgmt_og_menu_alter(&$items) {
       $item['title'] = isset($item['title']) ? $item['title'] : '';
       $item['title arguments'] = isset($item['title arguments']) ? $item['title arguments'] : array($item['title']);
       $item['title callback'] = isset($item['title callback']) ? $item['title callback'] : 't';
+
+      // If it's menu default local task and has missing access callback we use
+      // the callback from the parent.
+      if (($item['type'] == MENU_DEFAULT_LOCAL_TASK) && !isset($item['access callback'])) {
+        $parent_path = explode('/', $path);
+        array_pop($parent_path);
+        $parent_path = implode('/', $parent_path);
+        $parent = isset($items[$parent_path]) ? $items[$parent_path] : FALSE;
+        if ($parent) {
+          if (isset($parent['access callback'])) {
+            $item['access callback'] = $items[$parent_path]['access callback'];
+          }
+          if (!isset($item['access arguments']) && isset($parent['access arguments'])) {
+            $item['access arguments'] = $items[$parent_path]['access arguments'];
+          }
+        }
+      }
+
       $item['access arguments'] = isset($item['access arguments']) ? $item['access arguments'] : array();
```

Comment:
I checked the issue and what you report, and actually I would rather use this solution
https://www.drupal.org/node/553368 

$item['access arguments'] = isset($item['access arguments']) ? $item['access arguments'] : array(0);

Reply:
@Fefaine 
Please remember that a few lines below 0 will be replaced to 3 and with that approach, you check user access to "admin" path instead of pass user through 
`entity_access($op, $entity_type, $entity = NULL, $account = NULL)`

Reply:
Okééééé ! Actually there is another issue when the job is outside of og context, it's currently setting the argument to 6 .. 🤣 
So you are fixing this too, for me it's fine if QA approves.